### PR TITLE
[admin-tool] New admin command to register value schema into ZK directly

### PR DESCRIPTION
**Pull request by huangminchn**
New command:
admin-tool.sh --add-schema-into-zk --venice-zookeeper-url zk_url --cluster cluster --store storeName --value-schema-file file_path --value-schema-id newId

It will check whether the requested id is already registered, and will check whether the new schema is backward compatible with all existing value schemas.

## How was this PR tested?
CI in progress. Will also test the new command in EI.

## Does this PR introduce any user-facing changes?
- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

File: clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java [hunk 1]
```diff
@@ -1,102 +1,6 @@
...
-import static com.linkedin.venice.Arg.PARTITIONER_PARAMS;
-import static com.linkedin.venice.Arg.PARTITION_COUNT;
-import static com.linkedin.venice.Arg.PRINCIPAL;
-import static com.linkedin.venice.Arg.PROGRESS_INTERVAL;
-import static com.linkedin.venice.Arg.PUSH_ID;
-import static com.linkedin.venice.Arg.PUSH_STREAM_SOURCE_ADDRESS;
-import static com.linkedin.venice.Arg.READABILITY;
-import static com.linkedin.venice.Arg.READ_COMPUTATION_ENABLED;
-import static com.linkedin.venice.Arg.READ_QUOTA;
-import static com.linkedin.venice.Arg.REGIONS_FILTER;
```
Comment (ZacAttack) [inline clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java:72]: nix star import

File: clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java [hunk 2]
```diff
@@ -149,7 +53,9 @@ public enum Command {
       new Arg[] { URL, CLUSTER, SYSTEM_STORE_TYPE }
   ), SET_VERSION("set-version", "Set the version that will be served", new Arg[] { URL, CLUSTER, STORE, VERSION }),
   ADD_SCHEMA("add-schema", "", new Arg[] { URL, CLUSTER, STORE, VALUE_SCHEMA }),
-  ADD_DERIVED_SCHEMA("add-derived-schema", "", new Arg[] { URL, CLUSTER, STORE, VALUE_SCHEMA_ID, DERIVED_SCHEMA }),
+  ADD_SCHEMA_INTO_ZK(
+      "add-schema-into-zk", "", new Arg[] { VENICE_ZOOKEEPER_URL, CLUSTER, STORE, VALUE_SCHEMA, VALUE_SCHEMA_ID }
```
Comment (ZacAttack) [inline clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java:57]: add-schema-to-zk

File: clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java [hunk 3]
```diff
@@ -967,6 +975,43 @@ private static void applyValueSchemaToStore(CommandLine cmd) throws Exception {
...
+    String veniceZookeeperUrl = getRequiredArgument(cmd, Arg.VENICE_ZOOKEEPER_URL, Command.ADD_SCHEMA_INTO_ZK);
+    String valueSchemaFile = getRequiredArgument(cmd, Arg.VALUE_SCHEMA, Command.ADD_SCHEMA_INTO_ZK);
+    int valueSchemaId = Utils.parseIntFromString(
+        getRequiredArgument(cmd, Arg.VALUE_SCHEMA_ID, Command.ADD_SCHEMA_INTO_ZK),
+        Arg.VALUE_SCHEMA_ID.toString());
+    String valueSchemaStr = readFile(valueSchemaFile);
+    verifyValidSchema(valueSchemaStr);
+    Schema newValueSchema = Schema.parse(valueSchemaStr);
+
+    HelixSchemaAccessor schemaAccessor =
```
Comment (ZacAttack) [inline clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java:990]: This is actually tricky.  ZK will require SSL at some point, and today SSL is set up for helix objects via a JVM system argument.

Comment (ZacAttack) [reply clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java:990]: What we can probably do is require (or optionally require?) ssl arguments for keystore/truststore and then if they're present set up the jvm system variable before creating the necessary Helix objects.  That should make sure the SSL session is set up.

Comment (huangminchn) [reply clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java:990]: Oh, I see. Is SSL already enforced in ZK now? If so, I can make it a required argument.

Comment (huangminchn) [reply clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java:990]: @ZacAttack Could you please share the jvm variable names that I should check for? I couldn't find them in the code base.

File: clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java [hunk 4]
```diff
@@ -1,102 +1,6 @@
...
-import static com.linkedin.venice.Arg.URL;
-import static com.linkedin.venice.Arg.VALUE_SCHEMA;
-import static com.linkedin.venice.Arg.VALUE_SCHEMA_ID;
-import static com.linkedin.venice.Arg.VENICE_CLIENT_SSL_CONFIG_FILE;
-import static com.linkedin.venice.Arg.VERSION;
-import static com.linkedin.venice.Arg.VOLDEMORT_STORE;
-import static com.linkedin.venice.Arg.VSON_STORE;
-import static com.linkedin.venice.Arg.WRITEABILITY;
-import static com.linkedin.venice.Arg.WRITE_COMPUTATION_ENABLED;
+import static com.linkedin.venice.Arg.*;
```
Comment (ZacAttack) [inline clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java:3]: still nix star import.

**Review by ZacAttack — APPROVED** [review]
Looks good.

**Review by ZacAttack — APPROVED** [review]

**Pull request merged**